### DISCUSSION TITLE
protobuf and re2 code copied into ONNX Runtime

### DISCUSSION THREAD

**Discussion by woulamme** [microsoft/onnxruntime#22105] [Release Q&A] [answered]
Our vulnerability scanning tools are picking up on code from protobuf and re2 being included in ONNX Runtime. Since this code is copied into ONNX Runtime, our vulnerability scanner cannot infer the version of protobuf and re2 used. What versions of these did you include code from? Would be good to include this information in a file somewhere in your repo.

**Comment by mudream4869** [comment, marked as answer]
Protobuf: 21.12

Re2:
* Installed version (from cmake prefix path)
* If re2 does not exist, then download from github (2024-07-02 for main branch): https://github.com/microsoft/onnxruntime/blob/main/cmake/deps.txt#L53C5-L53C67
